### ISSUE TITLE
Training Hyper-parameter

### ISSUE THREAD

**Issue by SherlockHolmes221** [bobwan1995/PMFNet#21]
How did you set the training hyper-parameters(lr, lr_decay, max_iter...) and how to find out which is the best step when testing? (I think that the hyper-parameters you set are pretty well  and they play an important role in the final result.)
Looking forward to your reply.

**Comment by youngfly11** [comment]
Hi SherlockHolmes221:
As for the hyper-parameters, we conduct the sequential grid search for hyper-parameters on V-COCO val set. Once the hyper-parameters fixed, we select the last checkpoints to evaluate the test set.

**Comment by SherlockHolmes221** [comment]
Thanks a lot
